servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -38,14 +38,8 @@ def get_job_aggregations(timeFrame, projectId=None):
 
     try:
         for job in jobs:
-
             # TEST_TOKEN_VALIE is a global config value used to distinguish testing jobs from batch to batch by timestemp
-            if 'TEST_TOKEN_VALUE' in current_app.config and (
-                    'test_token' not in job.labels or job.labels['test_token']
-                    != current_app.config['TEST_TOKEN_VALUE']):
-                continue
-            # AGGREGATION_JOB_NAME_FILTER is a global config value used to distinguish aggregation testing jobs from others
-            if 'AGGREGATION_JOB_NAME_FILTER' in current_app.config and job.name != current_app.config['AGGREGATION_JOB_NAME_FILTER']:
+            if 'AGGREGATION_JOB_LABEL_FILTER' in current_app.config and job.labels['test_token'] != current_app.config['AGGREGATION_JOB_LABEL_FILTER']:
```

Comment:
Please don't hardcode `"test_token"` here, please see my original comment on proposed syntax, i.e. set this to `test_token=...`, then just parse that = out here. I realize there is exactly one user of this config parameter but still it is messy to depend on entering the exact same string here as in your test file.